packages/client/src/devices/InputMediaDeviceManager.ts

```diff
@@ -130,7 +130,9 @@ export abstract class InputMediaDeviceManager<
    */
   async select(deviceId: string | undefined) {
     if (isReactNative()) {
-      throw new Error('This method is not supported in React Native');
+      throw new Error(
+        'This method is not supported in React Native. Please visit http://localhost:3000/chat/docs/sdk/reactnative/core/camera-and-microphone/#speaker-management for reference.',
```

Comment:
```suggestion
        'This method is not supported in React Native. Please visit https://getstream.io/video/docs/reactnative/core/camera-and-microphone/#speaker-management for reference.',
```

We shouldn't have localhost in the error message :)

Reply:
Oops, I added this by mistake. Let me fix it.

Reply:
Fixed 👍🏻